### ISSUE TITLE
Publish modernized VA.gov a11y strategy document to platform website

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#46389]
## User Story
As an emerging accessibility practice at VA.gov, we want to publish our new strategy in an easily accessible (and publicly linkable) location to act as a north star to provide accountability and direction for all of our efforts.

As someone supporting VA.gov, I want to know how accessibility is measured and strategized across VA.gov so that I can be part of the solution for building a more accessible VA.gov.

Assignee: @briandeconinck 
Peer Reviewer: @CherylEvans 

## Description
We're looking to publish the [a11y strategy doc (google docs)](https://docs.google.com/document/d/1YJHAmuTWDLPpvYG-E1IKMmKyCv76o8PwHuU7hcPi_UA/edit?usp=sharing) to the platform website, but are still uncertain as to:
- what the process would look like
- where it would go
  - we've discussed if accessibility should be embedded (as part of research, design, engineering sections) or separate (as its own section)
  - my hunch at the moment is the [why not both taco meme](https://knowyourmeme.com/memes/why-not-both-why-dont-we-have-both) :taco: where the a11y strategy doc could live in its own section (where we can continue to add a11y-specific content like training materials) and we can point to it in the embedded sections like this [resource section in design](https://depo-platform-documentation.scrollhelp.site/research-design/#Researchanddesign-Accessibility)

## Impacted Artifacts
Depending on where we choose to publish this to, we may look into adding links to it within the research and design section of the platform website.
- [Draft document in Confluence](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2355921129/46389+-+Modernized+VA.gov+Accessibility+Strategy)

## Questions

> Should the entire document be published to Platform Website? Even the section prior to Mission? 

Everything but the reviewer section! We want to keep the context on the goal of this strategy doc and how it was created for transparency purposes 😄 

> Should it remain a Google doc or do we want the content embedded into a page of Platform Website?

We'd like for this content to be embedded into a page of Platform Website so it's a little bit more sticky, not in a permanent sense, but in a "this is published" sense.

> In the tasks below, I'm uncertain what, "Create a github doc of the VA.gov a11y strategy doc (for editing purposes if folks leave feedback)" - are you suggesting a duplicate GitHub doc? Would this be the doc that is edited by the non-Platform a11y specialists, and then regular updates are made to the Platform Website version?

We're hoping to shift editing and feedback away from the google doc and into a github version so updates can be made and maintained in one location.

> Platform Website includes a footer at the end of each page (see image below). This includes instructions for how VFS folks can suggest changes to the page. This may conflict with the other editing process above.

We hope to use the Platform Website method of editing, hence the transition away from google docs.

> Is there a list of other Platform Website pages where you'd like to add a reference link to this new page?

I'll create a list for that! Will also ask the other a11y specialists.

> How do we want to publish comments? Or do we want to publish comments?

We don't publish comments 😄 

> There may be things that are currently out of date in the Final Draft (for example, Sarah is no longer with DST). Should we fix now?

Yes, let's fix for now. Please leave comments on anything else that may be out of date in the google doc before we transition to github 🙏 

![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/d002c38d-fc37-40f7-9670-24a084b126bf)

## Tasks
- [x] Determine a location for the VA.gov a11y strategy doc on the platform website
   - [x] Reach out to Platform Content for recommendation
- [x] Publish on Platform website
- [ ] ~~Create a github doc of the VA.gov a11y strategy doc (for editing purposes if folks leave feedback)~~
   - This isn't a great solution for a11y folks editing, since transferring changes from GitHub to Confluence will be cumbersome. Feedback should be left via the link in the footer, and/or discussed informally with Brian.
- [ ] ~~Add links to the a11y strategy doc to other areas of the platform website (pending)~~
   - Adding links will be included in a separate ticket once the strategy doc is published on PW

## Peer Review
- [x] Ticket reassigned to peer reviewer when ready for peer review
- [ ] Peer Reviewer: Completes review and validates user story/Acceptance Criteria are met

## Acceptance Criteria
- [x] The Modernized VA.gov A11y Strategy document has been published to Platform Website and is available for VFS viewing

## How to prepare this issue
- [x] Attach story to appropriate Initiative or Quarterly Tech-Debt Epic.
- [x] Add User Story, description, tasks, and Acceptance Criteria
- [x] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [x] Once all above items are checked, add the 'Ready' label.
- [x] Ticket has been discussed as a team at Planning.
- [x] Ticket has been sized as a team.
- [x] Ticket has been assigned to a Sprint.
- [x] Once assigned to the upcoming or current Sprint, move to the Planned pipeline.
- [x] Once assigned to the upcoming or current Sprint, add assignee and specify Peer Reviewer.

**Comment by shiragoodman** [comment]
@joshkimux thanks so much for putting this together! My team plans to take it on this sprint, however we still have a lot of questions we'd like answered. I put a few of them in this doc (under the Questions heading) but it may be best to synch for 15-30 minutes with the larger group to really understand how you visualize this document being communicated on Platform Website. I'll reach out to you on Slack in the #accessibility-ops channel.

**Comment by joshkimux** [comment]
@shiragoodman left answers to all of these! Thank you so much!

**Comment by shiragoodman** [comment]
@joshkimux @naomimarcussen I removed you as the assignees and have added @briandeconinck (person who will do the work) and @claytonzook (peer reviewer). They will reach out if there are any questions/concerns.

**Comment by shiragoodman** [comment]
@CherylEvans I have added you as the peer reviewer for this (in place of Clayton). Please let me know if you have any questions!

**Comment by briandeconinck** [comment]
Posting a quick update here: Talked with @joshkimux about some clean-up to the doc, should be ready to publish in the next couple of days!

**Comment by briandeconinck** [comment]
@CherylEvans This should be ready to publish. Could you review the [draft in Confluence](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2355921129/46389+-+Modernized+VA.gov+Accessibility+Strategy) and make any edits as appropriate?

**Comment by CherylEvans** [comment]
@briandeconinck This doc looks really good!!  I had some minor changes that I made, but there were some questions and recommendations that I made whereby I didn't know if the changes I recommended would change the context.  They are all pretty straightforward.  Please feel free to reach out if you have any questions or concerns.  Also, I recommend making sure all the font styles throughout the doc are the same.  There was a lot that was italicized, and I wasn't sure if it was intentional.  Great job, by the way!!  Compiling all of that information  couldn't have been easy. :-)

**Comment by briandeconinck** [comment]
@CherylEvans [We're live on Platform website!](https://depo-platform-documentation.scrollhelp.site/getting-started/modernized-va-gov-accessibility-strategy) When you get a chance could you review the published version one last time? Then we can close this out.

**Comment by CherylEvans** [comment]
@briandeconinck I think **accessibility** and **strategy** should be capitalized in the title, as it's referring to the design of products, devices, services, vehicles, or environments so as to be usable by people with disabilities.  Also, according to the AP Stylebook: In title case, major words are capitalized, and most minor words are lowercase. In sentence case, most major and minor words are lowercase (proper nouns are an exception in that they are always capitalized).
major words: Nouns, verbs (including linking verbs), adjectives, adverbs, pronouns, and all words of four letters or more are considered major words.minor words: Short (i.e., three letters or fewer) conjunctions, short prepositions, and all articles are considered minor words.
Yes, I copy/pasted this!  I can't remember every single rule, so I always refer to my AP-SB.  :-) 
I think it looks great otherwise, and like I said, it is a lot of information to put out!

**Comment by briandeconinck** [comment]
Capitalized in Confluence, should publish the update today! I'll go ahead and move this to closed.

**Comment by CherylEvans** [comment]
@briandeconinck Thanks Brian!!

**Comment by briandeconinck** [comment]
@shiragoodman Just realized I forgot to tag you here, this is now closed!
